### DISCUSSION TITLE
How and when to migrate Connections Data to a new Connections Environment?

### DISCUSSION THREAD

**Discussion by canklamassono** [HCL-TECH-SOFTWARE/connections-automation#315] [Q&A]
Hello,

I have an old Connections 7 Server (without CP but with Docs) and am trying to migrate to my newly created Connections 8 Environment made by your amazing scripts.

What are the exact steps to take when trying to migrate all the data and when to do them?

Especially Docs is giving me permission problems - before migrating everything is fine - after migrating a Person can upload a file and afterwards isn't able to access it anymore.

Should Docs be installed after migrating for example?

**Comment by sabrina-yee** [comment]
Hi @canklamassono, I'd suggest to resolve this through a Support ticket so the issue can be analyzed in detail.  To me, it seems like this would be a side by side migration in which case, Docs can be installed before the migration and then proceed to migration the database and configurations.
